tests/setup.js

```diff
@@ -0,0 +1,20 @@
+const { mount, shallowMount } = require('@vue/test-utils');
+
+global.getMountUtils = ({
+  component,
+} = {}) => ({
```

Comment:
does this default param need to be here? It seems like `component` is required? (??)

Reply:
I just personally hate the error `cannot destructure <thing> from undefined` as it's super unhelpful